types/react-datepicker/react-datepicker-tests.tsx

```diff
@@ -1,30 +1,28 @@
-import * as React from "react";
+import * as React from 'react';
 import * as moment from 'moment';
-import * as DatePicker from 'react-datepicker';
+import DatePicker from 'react-datepicker';
 
-class ReactDatePicker extends React.Component<{}, { startDate: moment.Moment; displayName:string; }> {
-     constructor(props: {}) {
-        super();
-        this.state = {
-          startDate: moment(),
-          displayName: 'Example'
-        }
-        this.handleChange = this.handleChange.bind(this);
-     }
+class ReactDatePicker extends React.Component<{}, { startDate: moment.Moment; displayName: string; }> {
+	constructor(props: {}) {
+		super();
```

Comment:
The stylistic changes were because they were mandated by the linter.  It's the standard for DefinitelyTyped.